index.html

```diff
@@ -1,179 +1,250 @@
+              </section>
+            </section>
+            <section class="other">
+              <section class="informationCard">
+                <h3 class="subTitle">Multi-Post Stories Gaing+Glory</h3> 
+                <ul>
+                  <li class="myRecentWorks">Ruby on rails</li>
+                  <li class="myRecentWorks">css</li>
+                  <li class="myRecentWorks">JavaScript</li>
+                  <li class="myRecentWorks">html</li>
+                </ul>
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+            <section class="other">
+              <section class="informationCard">
+                <h3 class="subTitle">Multi-Post Stories Gaing+Glory</h3> 
+                <ul>
+                  <li class="myRecentWorks">Ruby on rails</li>
+                  <li class="myRecentWorks">css</li>
+                  <li class="myRecentWorks">JavaScript</li>
+                  <li class="myRecentWorks">html</li>
+                </ul>
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+            <section class="other">
+              <section class="informationCard">
+                <h3 class="subTitle">Multi-Post Stories Gaing+Glory</h3> 
+                <ul>
+                  <li class="myRecentWorks">Ruby on rails</li>
+                  <li class="myRecentWorks">css</li>
+                  <li class="myRecentWorks">JavaScript</li>
+                  <li class="myRecentWorks">html</li>
+                </ul>
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+            <section class="other">
+              <section class="informationCard">
+                <h3 class="subTitle">Multi-Post Stories Gaing+Glory</h3> 
+                <ul>
+                  <li class="myRecentWorks">Ruby on rails</li>
+                  <li class="myRecentWorks">css</li>
+                  <li class="myRecentWorks">JavaScript</li>
+                  <li class="myRecentWorks">html</li>
+                </ul>
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+            <section class="other">
+              <section class="informationCard">
+                <h3 class="subTitle">Multi-Post Stories Gaing+Glory</h3> 
+                <ul>
+                  <li class="myRecentWorks">Ruby on rails</li>
+                  <li class="myRecentWorks">css</li>
+                  <li class="myRecentWorks">JavaScript</li>
+                  <li class="myRecentWorks">html</li>
+                </ul>
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+          </div>  
+        </section>
```

Comment:
- [ ] [Optional] I am giving you an option because this is the last review for you. please implement it. Please decrease the spacing between tools and skills to stick to the design. See image below for reference
### Your design
![sssssssssssssss](https://user-images.githubusercontent.com/81727988/201500029-d09406ab-43fd-46d4-9183-3cd16829ebac.png)